dev/TitleBar/TitleBar.cpp

```diff
@@ -119,6 +119,11 @@ void TitleBar::OnCustomContentPropertyChanged(const winrt::DependencyPropertyCha
     UpdateHeight();
 }
 
+void TitleBar::OnTitlePropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    UpdateTitle(args.NewValue().as<winrt::hstring>());
```

Comment:
Instead of passing in the string, can we just reference Title in the update method? Either way certainly works but I feel it's cleaner for when UpdateTitle() is called from other places.

Reply:
Removed the param and calling Title() directly instead.